Initialization/linux-initialization-10.md

```diff
@@ -452,7 +452,7 @@ Links
 * [SLAB](http://en.wikipedia.org/wiki/Slab_allocation)
 * [xsave](http://www.felixcloutier.com/x86/XSAVES.html)
 * [FPU](http://en.wikipedia.org/wiki/Floating-point_unit)
-* [Documentation/security/credentials.txt](https://github.com/torvalds/linux/blob/16f73eb02d7e1765ccab3d2018e0bd98eb93d973/Documentation/security/credentials.txt)
+* [Documentation/security/credentials.txt](https://github.com/torvalds/linux/blob/master/Documentation/security/credentials.rst)
```

Comment:
Display text should be [/Documentation/security/credentials.rst]

Reply:
I would rather keep the pattern of these names the same, with no preceding '/'. Even though this is not an include directory, I believe the location is still obvious.